index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="fr" xml:lang="fr" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins:400,700,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h3 class="logo-text">My Logo</h3>
+      <img
+        class="menu"
+        src="./images/menu.svg"
+        alt="triangle with all three sides equal"
+        height="87"
+        width="100"
+      />
+    </header>
+    <main>
+      <h1 class="header-text">I'm Abraham Rodriguez Martinez to see you!</h1>
```

Comment:
- [ ] This texts are overlapping with the paragraph. Please fix that, 
- [ ] Also stick to the design for now I am seeing the website doesn't cover the whole page. Please fix that.

 Below is the screenshot.


![Screenshot from 2021-06-03 19-45-51](https://user-images.githubusercontent.com/22434660/120661683-1fad5300-c4a6-11eb-9274-9d5559954457.png)